pandas/tests/window/test_win_type.py

```diff
@@ -676,3 +676,13 @@ def test_cmov_window_special_linear_range(win_types_special, step):
         .mean(**kwds[win_types_special])
     )
     tm.assert_series_equal(xp, rs)
+
+
+@td.skip_if_no_scipy
+def test_weighted_var_big_window_no_segfault():
```

Comment:
Could you add the `win_types` fixture in the signature and assign `win_type=win_type`? Same with `center`

Reply:
> Same with `center`

I'm confused. There is no fixture for `centers` yet in `test_win_type.py`. Do you want me to add one? like:

```python
@pytest.fixture(params=[True, False])
def centers(request):
    return request.param
    
@td.skip_if_no_scipy
def test_weighted_var_big_window_no_segfault(win_types, centers):
    # Github Issue #46772
    x = Series(0)
    result = x.rolling(window=16, center=centers, win_type=win_types).var()
    expected = Series(np.NaN)

    tm.assert_series_equal(result, expected)
```

Reply:
This is defined in `confest.py` in the directory above, so pytest will grab that fixture if you add `center` in the test function signature.